### DISCUSSION TITLE
How to find repositories using LFS within an organization?

### DISCUSSION THREAD

**Discussion by phylor** [community/community#21954] [New to GitHub]
<p>Is there any way to find all repositories using Git LFS within an organization? And if so, is there also a way to know the disk quota per repository?</p>
<p>Thanks for any hints!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/phylor">@phylor</a> Thanks for being part of the Community Forum! This is a great question. The easiest way to find out exactly where LFS might be being used in an organization account that you own would actually be to reach out to private support. We can take a look at the details of your account and help you find this information, but we wouldn’t want to discuss your private account details in a public forum. You can reach us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> and even share the link to this forum discussion when writing in so you don’t have to restate your original question.</p>
<p>I hope this helps! Please let me know if I can help with anything else.</p>
